cdm/src/test/java/ucar/nc2/ft2/coverage/simpgeometry/TestGetGeometryType.java

```diff
@@ -0,0 +1,44 @@
+package ucar.nc2.ft2.coverage.simpgeometry;
+
+import java.io.IOException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import ucar.nc2.dataset.NetcdfDataset;
+import ucar.unidata.util.test.TestDir;
+
+/**
+ * Tests ability to get Simple Geometry Type using Simple Geometry Reader.
+ * 
+ * @author wchen@usgs.gov
+ *
+ */
+public class TestGetGeometryType {

```

Comment:
Test classes are conventionally named after the implementations that they are testing. Developers and IDEs tend to rely on this convention. Naming it something else could make it more difficult for developers and their IDEs to associate the files. Is there a strong argument for naming this class something other than `TestSimpleGeometryReader`?

Reply:
TestSimpleGeometryReader is more or less tested in the following classes:

- TestReadLine
- TestReadPoint
- TestReadPolygon
- TestGetGeometryType

We could possible squish these four test classes into gone TestSimpleGeometryReader class.

Reply:
Interesting! I managed to miss the spread :) My bad. Thanks for pointing it out. I would support you exploring putting the SimpleGeometryReader tests in one class. I wonder if some of the commonalities between the tests could be deduplicated along the way? If moving the tests into one class ends up being hard and takes us up close to the close of the sprint, I'm OK merging without this improvement.

Reply:
It shouldn't be too much work to get it done, will squish the tests into one class!